enum/technischeressourcenutzung/technischeressourcenutzung.go

```diff
@@ -0,0 +1,12 @@
+package technischeressourcenutzung
+
+//go:generate stringer --type Technischeressourcenutzung
+//go:generate jsonenums --type Technischeressourcenutzung
+
+type Technischeressourcenutzung int
```

Comment:
kannst du bitte das casing gleich machen wie in .net?
https://github.com/Hochfrequenz/BO4E-dotnet/blob/d3d89270f94672c5502eea9a632b9de1d7b7ac12/BO4E/ENUM/TechnischeRessourceNutzung.cs#L6

Reply:
ich mache es so wie in andere enums in diese repo

Reply:
Andere Enums sind auch CamelCase: https://github.com/Hochfrequenz/go-bo4e/blob/8e41ea5bb126a5a18f91a4dbbe6ad0e0e9ed533d/enum/abweichungsgrund/abweichungsgrund.go#L7

Reply:
ich habe  Netznutzungsabrechnungsgrundlage gesehen

Reply:
Ja eigentlich das ist ein deutsche Wort. aber besser ist wenn sie mit .net gleich sein.